apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -1,11 +1,48 @@
+import { AppRoute } from "~/libs/enums/enums.js";
+import { type ValueOf } from "~/libs/types/types.js";
+
+import { Link } from "../components.js";
+import styles from "./styles.module.css";
+
 type Properties = {
+	color?: "primary";
+	href?: ValueOf<typeof AppRoute>;
+	isFluid?: boolean;
+	isSmall?: boolean;
```

Comment:
```suggestion
	size?: 'small' | regular;
```
default is `regular`